### ISSUE TITLE
Remove doc/core/upgrades

### ISSUE THREAD

**Issue by twisted-trac** [twisted/twisted#5484]
|[<img alt="thijstriemstra's avatar" src="https://avatars.githubusercontent.com/u/305679?s=50" width="50" height="50">](https://github.com/thijstriemstra)| @thijstriemstra reported|
|-|-|
|Trac ID|trac#5484|
|Type|task|
|Created|2012-02-13 17:43:58Z|
|Branch|https://github.com/twisted/twisted/tree/upgrades-doc-5484|

The documentation in [doc/core/upgrades](doc/core/upgrades) describes the zope.interface migration and twisted split. All this describes things from 5 years or more ago, so it's not relevant any more (to new users).

<details><summary>Searchable metadata</summary>

```
trac-id__5484 5484
type__task task
reporter__thijs thijs
priority__normal normal
milestone__ 
branch__branches_upgrades_doc_5484 branches/upgrades-doc-5484
branch_author__thijs thijs
status__closed closed
resolution__fixed fixed
component__core core
keywords__documentation__easy documentation, easy
time__1329155038000000 1329155038000000
changetime__1331143912000000 1331143912000000
version__None None
owner__thijs thijs
cc__thijs cc__jesstess
```
</details>

**Comment by twisted-trac** [comment]
|[<img alt="thijstriemstra's avatar" src="https://avatars.githubusercontent.com/u/305679?s=50" width="50" height="50">](https://github.com/thijstriemstra)|@thijstriemstra set owner to @thijstriemstra<br>@thijstriemstra set status to `assigned`|
|-|-|

**Comment by twisted-trac** [comment]
|[<img alt="thijstriemstra's avatar" src="https://avatars.githubusercontent.com/u/305679?s=50" width="50" height="50">](https://github.com/thijstriemstra)<a name="note_2"></a>|@thijstriemstra commented|
|-|-|

(In [33618]) Branching to 'upgrades-doc-5484'

**Comment by twisted-trac** [comment]
|[<img alt="thijstriemstra's avatar" src="https://avatars.githubusercontent.com/u/305679?s=50" width="50" height="50">](https://github.com/thijstriemstra)<a name="note_3"></a>|@thijstriemstra commented|
|-|-|

(In [33619]) remove 2.0 upgrade docs, refs [#5484](https://github.com/twisted/twisted/issues/5484)

**Comment by twisted-trac** [comment]
|[<img alt="thijstriemstra's avatar" src="https://avatars.githubusercontent.com/u/305679?s=50" width="50" height="50">](https://github.com/thijstriemstra)|@thijstriemstra removed owner<br>@thijstriemstra set status to `new`|
|-|-|

**Comment by twisted-trac** [comment]
|[<img alt="jesstess's avatar" src="https://avatars.githubusercontent.com/u/188336?s=50" width="50" height="50">](https://github.com/jesstess)|@jesstess set owner to @jesstess|
|-|-|

**Comment by twisted-trac** [comment]
|[<img alt="jesstess's avatar" src="https://avatars.githubusercontent.com/u/188336?s=50" width="50" height="50">](https://github.com/jesstess)<a name="note_6"></a>|@jesstess set owner to @thijstriemstra|
|-|-|

Thanks for working on this, thijs.

Your changes look good to me, and [buildbot](http://buildbot.twistedmatrix.com/boxes-supported?branch=/branches/upgrades-doc-5484) is fine, so please merge!

Tangential to this ticket, the `development` subdirectory of http://twistedmatrix.com/documents/current/core/ is missing from `test_coreProjectLayout`.

**Comment by twisted-trac** [comment]
|[<img alt="thijstriemstra's avatar" src="https://avatars.githubusercontent.com/u/305679?s=50" width="50" height="50">](https://github.com/thijstriemstra)<a name="note_7"></a>|@thijstriemstra set status to `closed`|
|-|-|

(In [33660]) Merge upgrades-doc-5484: Remove the obsolete documentation for upgrading to Twisted Core 2.0 and newer.

Author: thijs
Reviewer: jesstess
Fixes: [#5484](https://github.com/twisted/twisted/issues/5484)
